docs/cedarling/cedarling-policy-store.md

```diff
@@ -10,59 +10,68 @@ tags:
 
 # Cedarling Policy Store
 
-By convention, the filename is `cedarling_store.json`. It contains all the data the 
-Cedarling needs to evaluate policies and verify JWT tokens:
+The Cedarling Policy Store uses a JSON file named `cedarling_store.json` to store all necessary data for evaluating policies and verifying JWT tokens. The structure includes the following key components:
 
-1. Cedar Schema - Base64 encoded human format
-2. Cedar Policies - Base64 encoded human format
-3. Trusted Issuers - See below syntax
+1. **Cedar Schema**: The Cedar schema encoded in Base64.
+2. **Cedar Policies**: The Cedar policies encoded in Base64.
+3. **Trusted Issuers**: Details about the trusted issuers (see [below](#trusted-issuer-schema) for syntax).
 
-The JSON schema looks like this:
+**Note:** The `cedarling_store.json` file is only needed if the bootstrap property, `CEDARLING_POLICY_STORE_URI`, is not set to a local location. If it is a non-local location, Cedarling will fetch the policy store from the URI.
 
-```
+### JSON Schema Example
+
+Here is a non-normative example of a `cedarling_store.json` file:
+
+```json
 {
-    "some_unique_identifier": {
-        "name": "...",
-        "description": "...",
-        "policies": {},
-        "identity_source": {},
-        "schema": "..."
-    }
+    "cedar_version": "v2.4.7",
+    "cedar_policies": {
+        "unique_policy_id": {
+            "description": "simple policy example",
+            "creation_date": "2024-09-20T17:22:39.996050",
+            "policy_content": "cedarling_policy_encoded_in_base64"
+        }
+    },
+    "cedar_schema": "cedar_schema_encoded_in_base64",
+    "trusted_issuers": [
+        {
+            "name": "IDP1",
+            "description": "some idp",
+            "openid_configuration_endpoint": "https://www.idp.com/.well-known/openid-configuration",
+            "token_metadata": [
+                {
+                    "type": "Access",
+                    "person_id": "aud"
```

Comment:
this should be `user_id`. The token type are `access`, `id_token` and `userinfo`.

Reply:
> this should be user_id (not person_id)

It's the claim used to create the person entity. Thus, think it's better to call it `person_id`.

> The token type are access, id_token and userinfo.

why have the `_token` suffix for `id_token` but not `access_token` and `userinfo_token`?

Reply:
i just made it so they need to pass either `id_token`, `access_token`, `userinfo_token`, or `transaction_token` so it's more consistent. see: [a5abfd9](https://github.com/JanssenProject/jans/pull/9910/commits/a5abfd992ac2c73b1878df5b21b9d782bd7b1275)

Reply:
please discuss your POV in a thread and get approval.

Reply:
The discussion has been made and it has been agreed upon to use the following convention:
- `access_token`
- `id_token`
- `userinfo_token`
- `trasaction_token`

Reply:
all instances of `person_id` has also been changed to `user_id` here: [d50c9d9](https://github.com/JanssenProject/jans/pull/9910/commits/d50c9d9857242ef19f99fa6749e664e4cc3373db)